src/main/java/org/jenkins/tools/test/model/hook/PluginCompatTesterHooks.java

```diff
@@ -179,7 +182,7 @@ private Map<String, Queue<PluginCompatTesterHook>> findHooks(String stage) {
             // Ignore abstract hooks
             if (!Modifier.isAbstract(c.getModifiers())) {
                 try {
-                    System.out.println("Hook: " + c.getName());
+                    LOGGER.log(Level.FINE, "Loading hook: {0}", c.getName());
```

Comment:
Similarly, this was creating a lot of log noise, one statement for each hook, even if the hook never got executed. Lowering this to `FINE` level since the typical user shouldn't care about hooks that aren't running; rather, this is relevant when debugging something while developing a hooks-related change.